bnelson777/id.ly
# IL-113: Get bluetooth package running

# Jira Issues this branch covers
Il-113: Get Bluetooth package running
Il-143: Get ios bluetooth working
Il-146: Use device name instead of ID for BT
Il-88: Some kind of confirmation that an identity has been added after successful scan
Il-141: Encrypt card sent over bluetooth using AES

# Testing
* Load app into both device 1 and device 2
* Share a card of device 1
* Scan a card on device 2
* Press the "acquire" button when it pops up
* Ensure that device 1's card is in device 2's Rolodex
* Try multiple transactions and swapping cards
